sqlalchemy_history/utils.py

```diff
@@ -400,3 +401,14 @@ def get_association_proxies(klass):
         if isinstance(prop, sa.ext.associationproxy.AssociationProxy):
             association_proxy_mapping[key] = prop
     return association_proxy_mapping
+
+def datetime_utc_now():
+    """
+    Python 3.12 onwards datetime.datetime.utcnow is deprecated and recommended usage
+    is to use datetime.datetime.now(datetime.UTC)
+    ref: https://docs.python.org/3/library/datetime.html#datetime.datetime.utcnow
+    """
+    try:
+        return datetime.datetime.now(datetime.UTC)
+    except AttributeError: # python <3.12
```

Comment:
datetime.UTC was added in py3.11
But it is just an alias for datetime.timrzone.utc


So instead of try except if you use

datetime.datetime.now(datetime.timezone.utc)

It will be compatible with all python3.x versions and you don't need this until function